codegen/build-oas.sh

```diff
@@ -3,7 +3,7 @@
 set -eux -o pipefail
 
 version=$1 # e.g. 2024-07
-modules=("control")
+modules=("db_control" "db_data" "inference")
```

Comment:
1. replaced control with db_control 
2. added db_data for bulk import (actual feature is going to be a part of the next PR).
3. added inference which is now broken out of control plane